resource-managers/mesos/src/test/scala/org/apache/spark/scheduler/cluster/mesos/MesosClusterSchedulerSuite.scala

```diff
@@ -48,45 +48,50 @@ class MesosClusterSchedulerSuite extends SparkFunSuite with LocalSparkContext wi
     }
 
     driver = mock[SchedulerDriver]
-    scheduler = new MesosClusterScheduler(
-      new BlackHoleMesosClusterPersistenceEngineFactory, conf) {
-      override def start(): Unit = { ready = true }
+    val persistenceFactory = new BlackHoleMesosClusterPersistenceEngineFactory()
+    scheduler = new MesosClusterScheduler(persistenceFactory, conf) {
+      override def start(): Unit = { this.ready = true }
     }
     scheduler.start()
+    scheduler.registered(driver, Utils.TEST_FRAMEWORK_ID, Utils.TEST_MASTER_INFO)
+
+    verify(driver, times(1)).suppressOffers()
```

Comment:
Here this call has no result right, no suppress.We add the driver and then at the end this called is made, so the queue is not empty, correct?